### PR TITLE
Sad test/login

### PR DESCRIPTION
**Pull request by JoshMallery**
##Sad Path Testing for Login and Homepage
Limited testing for invalid inputs on the login page for a user and for a user to login. 

#### Type of Change:
- [] Bug fix (non-breaking change which fixes an issue)
- [] Refactor (code is working and without bugs, formatting has been streamlined)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?
What steps have you taken to ensure the code is effective?
- [] Have you used Dev Tools/ ran code in the console?
- [x] Have you ran the Cypress testing suite?

#### Checklist:
- [x] My code follows the style guidelines of this project.
- [x] My commit messages are consistent, descriptive, concise and begin with a verb and capital letter.
- [x] I have performed a self-review of my own code.
- x[] I have added reviewers.

### CODE REVIEW

File: cypress/e2e/homepage-flow.cy.js [hunk 1]
```diff
@@ -129,4 +129,112 @@ describe('Homepage', () => {
...
+        })
+      } else {
+        console.log('Not working')
+      }
+    })
+
+
+    cy.get('[placeholder="Username"]').type('dan')
+    cy.get('[placeholder="Email"]').type('dan@gmail.com')
+    cy.get('.button').click()
```
Comment (DanielN88) [inline cypress/e2e/homepage-flow.cy.js:190]: Look how sad all these paths are. Everything looks good.

**Review by DanielN88 — APPROVED** [review]
Ready to merge

File: cypress/e2e/homepage-flow.cy.js (hunk 1 above)
Comment (aominhlong) [reply cypress/e2e/homepage-flow.cy.js:190]: Great job!!

**Comment by aominhlong** [comment]
Good working testing for sad paths!

**Review by aominhlong — APPROVED** [review]

File: cypress/e2e/homepage-flow.cy.js [hunk 2]
```diff
@@ -129,4 +129,112 @@ describe('Homepage', () => {
...
+          res.body.data.fetchUser.dogs = [
+            {
+                id: '2',
+                name: 'Dany',
+                age: 4,
+                breed: 'Mix',
+                skills: []
+              }
+          ]
+          console.log('Working')
```
Comment (Jlavarine) [inline cypress/e2e/homepage-flow.cy.js:180]: Great job testing for sad paths too!

**Review by Jlavarine — APPROVED** [review]

**Pull request merged**